src/test/java/org/openelisglobal/address/PersonAddressTest.java

```diff
@@ -0,0 +1,46 @@
+package org.openelisglobal.address;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.address.valueholder.PersonAddress;
+
+public class PersonAddressTest {
+
+    private PersonAddress personAddress;
+
+    @Before
+    public void setUp() {
+        personAddress = new PersonAddress();
+    }
+
+    @Test
+    public void testSetAndGetPersonId() {
+        personAddress.setPersonId("123");
```

Comment:
This can be done like this

`Assert.assertEquals(personAddress.setPersonId("123"),  personAddress.getPersonId());`

Reply:
Thanks @tendomart for review.
 when I do it like that I get error.

Reply:
oh okk.. did you asign the value somewhere, probably in a global variable   `personAddress` 

so all other methods can use it?

Reply:
No, I did not do that but I do commit now to do that, can you review it?